[Hung.vo]-BaiTap/index.html

```diff
@@ -0,0 +1,223 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="shortcut icon" type="image/png" href="image/favicon.ico" />
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="css/FlexRow.css">
+    <link rel="stylesheet" href="css/reponsive.css">
+
+    <title>Bài tập</title>
+</head>
+
+<body>
+    <div id="page-wrapper">
+        <!-- Header -->
+        <section id="header">
+            <div class="container navbar-expand-lg">
+                <div id="menu" class="row">
+                    <div class="col-12">
+                        <h1><a href="index.html" id="logo">Halcyonic</a></h1>
+                        <nav id="nav">
+                            <a href="index.html">Homepage</a>
+                            <a href="page/threecolumn.html">Three Column</a>
+                            <a href="page/twocolumn1.html">Two Column #1</a>
+                            <a href="page/twocolumn2.html">Two Column #2</a>
+                            <a href="page/onecolumn.html">One Column</a>
+                        </nav>
+
+                    </div>
+                </div>
+            </div>
+            <div id="banner">
+                <div class="container">
+                    <div class="row">
+                        <div class="col-6 col-12-medium">
+                            <p>We do something really useful, important, and unique. Learn all about it here ...</p>
+                            <a href="#" class="button-large">Go on, click me!</a>
+
+                        </div>
+                        <div class="col-6 col-12-medium imp-medium">
+                            <a href="#" class="bordered-feature-image"><img src="image/banner.jpg" alt="" /></a>
+
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section id="features">
+            <div class="container">
+                <div class="row">
+                    <div class="col-3 col-6-medium col-12-small">
+                        <section>
+                            <a href="#" class="bordered-feature-image"><img src="image/pic01.jpg" alt="" /></a>
```

Comment:
Đặt lại tên ảnh cho có nghĩa
vd: welcome-to....jpg

tránh đặt tên 01, 02 nếu đó không phải là ảnh tự updload